### PR TITLE
fix(bigtable): multiple region tags on same line

### PR DESCRIPTION
**Pull request by billyjacobson**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/6051)

### CODE REVIEW

**Review by coryan — APPROVED** [review]
LGTM.  I am going to ignore the conventional commits warning.

File: google/cloud/bigtable/examples/table_admin_snippets.cc [hunk 1]
```diff
@@ -292,7 +293,8 @@ void GetFamilyMetadata(google::cloud::bigtable::TableAdmin const& admin,
     std::cout << "Column family metadata for <" << family_name << "> is "
               << pos->second.DebugString() << "\n";
   }
-  // [END bigtable_get_family_metadata] [END bigtable_get_family]
+  // [END bigtable_get_family_metadata]
+  // [END bigtable_get_family]
```
Comment (devbww) [inline google/cloud/bigtable/examples/table_admin_snippets.cc:297]: Nit: The non-nested ordering strikes my nested eye.

Comment (coryan) [reply google/cloud/bigtable/examples/table_admin_snippets.cc:297]: @billyjacobson thoughts?

Comment (billyjacobson) [reply google/cloud/bigtable/examples/table_admin_snippets.cc]: good catch, fixed

**Review by coryan — APPROVED** [review]

**Comment by coryan** [comment]
Thanks!

**Pull request merged**